powershell/resources/psruntime/BuildTime/Cmdlets/ExportProxyCmdlet.cs

```diff
@@ -105,6 +105,7 @@ protected override void ProcessRecord()
 
                 Directory.CreateDirectory(variantGroup.OutputFolder);
                 File.WriteAllText(variantGroup.FilePath, sb.ToString());
+                File.AppendAllText(Path.Combine(variantGroup.OutputFolder, "file_contain_all_scripts.ps1"), sb.ToString());
```

Comment:
- Would be nice to add a line of vertical space in between the cmdlets.

Also, let's name the file `ProxyCmdletDefinitions.ps1` or something similar

Reply:
Will make the change